sql/src/main/java/io/crate/action/sql/SessionContext.java

```diff
@@ -38,6 +39,7 @@
 
     private SearchPath searchPath;
     private boolean hashJoinEnabled = true;
+    private Set<String> excludedOptimizerRules;
```

Comment:
I would have prefer to have something more typesafe than the full class name of the rule.